tests/system-tests/test-cases/beforeFlow-test.js

```diff
@@ -61,9 +61,10 @@ describe('Before test flow', function () {
         console.log('REPORT:', testReport);
```

Comment:
testReport is going to be empty since it's the final report that is returned from artillery so can delete these console.logs from all the system tests

Reply:
done